cypress/cypress.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
Just to be sure and to continue the conversation from the other thread:

> However, the setCommonConfig will only contain logic that's currently in stage/composer plugin.js - as I explained static cypress.json is still required, so I would like to keep all static configuration in one place instead of having some configuration options set in cypress.json, and some in setCommonConfig.

After considering it for a while, I reckon that it makes sense to do it this way around :smile: I was worried about not being able to override the individual settings in Stage, but we could always write a custom plugin that would do it, and that plugin would take precedence over the config file :slightly_smiling_face: 

Thus, all is good here :slightly_smiling_face: I'm just posting the comment to give my thoughts